shared/chat/conversation/list-area/normal/index.desktop.js

```diff
@@ -90,8 +104,13 @@ class Thread extends React.PureComponent<Props, State> {
 
     // someone requested we scroll down
     if (this.props.listScrollDownCounter !== prevProps.listScrollDownCounter) {
-      this.setState(p => (p.isLockedToBottom ? null : {isLockedToBottom: true}))
```

Comment:
Why remove this state?

Reply:
It isn't true in the general case, seems necessary for this PR.

Reply:
O yeah, I missed that this currently doesn't even do anything

Reply:
Yeah. Also, I think that the flag is automagically maintained by the scroll handlers.